PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -40,18 +40,15 @@
     <AssemblyName>CSharpPluginPayload</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
-    <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\SonarQube.MSBuild.PreProcessor.exe" />
+    <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\MSBuild.SonarQube.Internal.PreProcess.exe" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\SupportedBootstrapperVersions.xml" />
-    <FilesToCopy Include="$(SourcesRoot)\SonarQube.TeamBuild.PreProcessor\bin\$(Configuration)\SonarQube.MSBuild.PreProcessor.exe.config" />
```

Comment:
Unrelated to the renaming: I've dropped the stub three .config files (pre-processor, post-processor and SonarRunner shim). Was there are particular reason we included these?